### PR TITLE
Swat input rewrite

### PR DESCRIPTION
**Pull request by Isgotkowitz**
This PR rewrites the input files needed to run the SWAT simulation with the `InputManager`.

### CODE REVIEW

File: input/fertilizer_schedule/swat_fertilizer_schedule.json [hunk 1]
```diff
@@ -38,5 +38,7 @@
     "phosphorus_masses": [0,0,0,0],
     "potassium_masses": [0,0,0,0],
     "application_depths": [0.0, 0.0, 0.0, 0.0],
-    "surface_remainder_fractions": [1.0, 1.0, 1.0, 1.0]
+    "surface_remainder_fractions": [1.0, 1.0, 1.0, 1.0],
```
Comment (matthew7838) [inline input/fertilizer_schedule/swat_fertilizer_schedule.json]: Good eye

**Review by matthew7838 — APPROVED** [review]
LGTM, great work.

File: input/fertilizer_schedule/swat_fertilizer_schedule.json (hunk 1 above)
Comment (Isgotkowitz) [reply input/fertilizer_schedule/swat_fertilizer_schedule.json]: Thanks:)

**Review by ltp23 — APPROVED** [review]
Looks good to me, too. Thanks Ed!

**Pull request merged**